.gitignore

```diff
@@ -10,11 +10,6 @@ torch_xla/csrc/version.cpp
 *.pyc
 *.so
 
-# Files autogenerated by scripts/generate_code.sh
```

Comment:
Just curious -- why are we able to remove this? IIRC, we generate our xla ops each build. Do our existing ops still get generated if we remove this?